docs/customizing_deis/router_settings.rst

```diff
@@ -92,3 +94,22 @@ Be sure that your custom image functions in the same way as the `stock router im
 Deis. Specifically, ensure that it sets and reads appropriate etcd keys.
 
 .. _`stock router image`: https://github.com/deis/deis/tree/master/router
+
+PROXY Protocol
+---------------
+The PROXY Protocol is a simple protocol supported by nginx; HAProxy; Amazon ELB and others, and
+provides a method to obtain information about the original requests IP address sent to a load
+balancer in front of deis :ref:`router`.
```

Comment:
**Deis** should be capitalized in general unless referring to the command-line client.